dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlanner.cs

```diff
@@ -59,7 +59,11 @@ public class StepwisePlanner : IStepwisePlanner
         this._promptConfig = this.Config.PromptUserConfig ?? LoadPromptConfigFromResource();
 
         // Set MaxTokens for the prompt config
-        this._promptConfig.Completion.MaxTokens = this.Config.MaxTokens;
+        if (this._promptConfig.Completion is null)
+        {
+            this._promptConfig.Completion = new AI.AIRequestSettings();
```

Comment:
I think using `AI` namespace explicitly is not required here, we can add `using Microsoft.SemanticKernel.AI;` at the top and use just `AIRequestSettings`.